lib.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "@reason-native/lib",
```

Comment:
We should not name this "lib" that's too broad of a name for what this code is. It should be experimental and slowly upgraded out to it's own repo that can be depended on as it's cleaned up. A lot of the code in here is poorly documented, incomplete, and written quite a while ago. (I'm allowed to trash on my own code 😆 ).

Perhaps `@reason-native/do-not-use-this-will-break` or something

Reply:
Good idea! I've given it a ridiculous name haha :')

Reply:
"@reason-native/do-not-use-this-will-break"
Approach one:
Publishing the utilities as package: Easier from a build/contributor standpoint. The problem with this approach is when you depend on both `rely` and `refmterr`, and then you bump the version of `rely` which happened to bump its dependency version on the utils package, then you're forced to also bump your version of `refmterr`.

Approach two:
Vendoring it multiple times (just copy/paste the code between the two directories). That will make it slightly slower to build/install both `rely` and `refmterr` in one project because `rely` and `refmterr` would both build their vendored utilities.

The package manager concerns in approach one are not that bad because these are both only ever `devDependencies` and package manager troubles get multiplied when they're downstream (non-devDependencies). So at least the trouble is limited with Approach one.
For now, I'm kind of inclined just to suggest Approach Two. Thoughts?

Reply:
@jordwalke I'm not sure how well this would play out on Windows, but what if we leave the source where it is but instead symlink to it from within the individual subjects?